terraform/trial_rg/services.tf

```diff
@@ -59,8 +59,8 @@ module "trial_app_service_practitioner" {
     "EUREKA_CLIENT_SERVICEURL_DEFAULTZONE"    = "${module.trial_sc_discovery.hostname}/eureka/"
     "EUREKA_INSTANCE_HOSTNAME"                = "${local.practitioner_name}.azurewebsites.net"
     "FHIR_URI"                                = module.fhir_server.hostname
-    "ROLE_SERVICE_URI"                        = module.trial_app_service_role.hostname
-    "SITE_SERVICE_URI"                        = module.trial_app_service_site.hostname
+    "ROLE_SERVICE_URI"                        = "http:/role-service"
```

Comment:
why is this change needed? I thought you can just delete these 2 lines

Reply:
We need the service name. The difference is that it's hard coded. Decided for now to leave with the old mechanisem and when refactoring the client to change it (Instead of changing it everywhere in the code as it is today).
WDYT?